Cassiopee/Connector/Connector/IBM.py

```diff
@@ -227,8 +227,6 @@ def prepareIBMData(t_case, t_out, tc_out, t_in=None, to=None, tbox=None, tinit=N
         Reynolds = Internal.getValue(Reynolds)
         if Reynolds < 1.e5: frontType = 1
     else: Reynolds = 1.e6
-
-    if frontType == 42: expand= 4
```

Comment:
Does F42 still require expand 4 if the offset refinement is not used? If so, could this if statement be modified such that expand=4 is still set in the code if the offset refinements are not provided?